docs/user-guide/openai-api/intro.md

```diff
@@ -0,0 +1,80 @@
+---
+sidebar_position: 1
+---
+
+# Start an LlamaEdge API service
+
+Since LlamaEdge provides an OpenAI-compatible API service, it can be a drop-in replacement for OpenAI in almost all LLM applications and frameworks. 
+Checkout the articles in this section for instructions and examples for how to use locally hosted LlamaEdge API services in popular LLM apps.
+
+But first, you will need to start an [LlamaEdge API server](https://github.com/LlamaEdge/LlamaEdge/tree/main/api-server). But the steps are a little different from just a chatbot.
+
+## Step 1: Install WasmEdge
+
+```
+curl -sSf https://raw.githubusercontent.com/WasmEdge/WasmEdge/master/utils/install_v2.sh | bash -s
+```
+
+## Step 2: Download an LLM model
+
+```
+curl -LO https://huggingface.co/second-state/Llama-3-8B-Instruct-GGUF/resolve/main/Meta-Llama-3-8B-Instruct-Q5_K_M.gguf
+```
+
+## Step 3: Download an embedding model
+
+```
+curl -LO https://huggingface.co/second-state/All-MiniLM-L6-v2-Embedding-GGUF/resolve/main/all-MiniLM-L6-v2-ggml-model-f16.gguf
+```
+
+It is used by many agent and RAG apps to convert text-based knowledge into vectors for easy search and retrieval.
+
+## Step 4: Start the API server!
+
+```
+wasmedge --dir .:. \
```

Comment:
If I just want to create an API server for the LLM, why i need to run an embedding model?